sample/src/androidTest/java/com/schibsted/spain/barista/sample/introduction/IntroducingBaristaActions.java

```diff
@@ -85,7 +90,7 @@ public void letsIntroduceBarista() {
     openDrawer(R.id.drawer);
     closeDrawer(R.id.drawer);
 
-    //Interact with SwipeRefreshLayout (pullToRefresh)
+    // Pull to refresh a SwipeRefreshLayout
```

Comment:
Thanks for improving the code! 
But for another time please try to limit your changes to the ones related to your PR ^^' 🙏